### PR TITLE
Adding ewmMean, ewmStd, ewmVar

### LINKED ISSUE #126: Add ewm
Would be amazing to see exponential weighted momentum functions in this library: mean, std.

https://pola-rs.github.io/polars/py-polars/html/reference/series/api/polars.Series.ewm_mean.html

### PR DESCRIPTION
**Pull request by Bidek56**
Adding ewmMean, ewmStd, ewmVar to close #126

### CODE REVIEW

**Review by universalmind303 — CHANGES_REQUESTED** [review]
since all of the series functions use the expr functions & should be identical, we should create a new interface in `shared-traits` similar to what has been done for other shared functions.

File: __tests__/series.test.ts [hunk 1]
```diff
@@ -424,120 +424,142 @@ describe("series", () => {
...
-    ${"rollingMax"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingMax(2)}                                                | ${pl.Series("", [null, 2, 3, 3, 2], pl.Float64)}
-    ${"rollingMin"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingMin(2)}                                                | ${pl.Series("", [null, 1, 2, 2, 1], pl.Float64)}
-    ${"rollingSum"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingSum(2)}                                                | ${pl.Series("", [null, 3, 5, 5, 3], pl.Float64)}
-    ${"rollingMean"}     | ${pl.Series([1, 2, 3, 2, 1]).rollingMean(2)}                                               | ${pl.Series("", [null, 1.5, 2.5, 2.5, 1.5], pl.Float64)}
-    ${"rollingVar"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingVar(2)[1]}                                             | ${0.5}
-    ${"rollingMedian"}   | ${pl.Series([1, 2, 3, 3, 2, 10, 8]).rollingMedian({ windowSize: 2 })}                      | ${pl.Series([null, 1.5, 2.5, 3, 2.5, 6, 9])}
+    name                 | actual                                                                                         | expected
```
Comment (universalmind303) [inline __tests__/series.test.ts:427]: I think we can just throw all of the tests in `expr.test`

This is what we are doing with other ones. (testing series + expr) at the same time

example: 
```ts
  test("contains:regex", () => {
    const df = pl.DataFrame({
      a: ["Foo", "foo", "FoO"],
    });

    const re = new RegExp("foo", "i");
    const expected = pl.DataFrame({
      a: ["Foo", "foo", "FoO"],
      contains: [true, true, true],
    });
    const seriesActual = df.getColumn("a").str.contains(re).rename("contains");
    const actual = df.withColumn(col("a").str.contains(re).as("contains"));
    expect(actual).toFrameEqual(expected);
    expect(seriesActual).toSeriesEqual(expected.getColumn("contains"));
  });

File: polars/lazy/expr/index.ts [hunk 2]
```diff
@@ -140,6 +140,210 @@ export interface Expr
...
+   *         :math:`(1-\alpha)^2` and :math:`1` if ``adjust=true``, and
+   *         :math:`(1-\alpha)^2` and :math:`\alpha` if ``adjust=false``.
+   *       - When ``ignoreNulls=true``, weights are based
+   *         on relative positions. For example, the weights of
+   *         :math:`x_0` and :math:`x_2` used in calculating the final weighted
+   *         average of [:math:`x_0`, None, :math:`x_2`] are
+   *         :math:`1-\alpha` and :math:`1` if ``adjust=true``,
+   *         and :math:`1-\alpha` and :math:`\alpha` if ``adjust=false``.
+
+   * Examples
```
Comment (universalmind303) [inline polars/lazy/expr/index.ts:181]: i dont think tsdoc supports mathjax.

File: polars/lazy/expr/index.ts [hunk 3]
```diff
@@ -140,6 +140,210 @@ export interface Expr
...
+   *         recursively by
+   *         .. math::
+   *           y_0 &= x_0 \\
+   *           y_t &= (1 - \alpha)y_{t - 1} + \alpha x_t
+   * bias
+   *   When ``bias=False``, apply a correction to make the estimate statistically unbiased.
+   * minPeriods
+   *   Minimum number of observations in window required to have a value
+   *   (otherwise result is null).
+   * ignoreNulls
```
Comment (universalmind303) [inline polars/lazy/expr/index.ts:165]: refactor to JS/TS doc standards

File: polars/lazy/expr/index.ts [hunk 4]
```diff
@@ -140,6 +140,210 @@ export interface Expr
...
```
Comment (universalmind303) [inline polars/lazy/expr/index.ts:233]: refactor to TS/JSdoc syntax.

File: polars/lazy/expr/index.ts [hunk 5]
```diff
@@ -140,6 +140,210 @@ export interface Expr
...
```
Comment (universalmind303) [inline polars/lazy/expr/index.ts:301]: same as above

File: polars/series/index.ts [hunk 6]
```diff
@@ -222,6 +222,228 @@ export interface Series
    * Create a new Series that copies data from this Series without null values.
    */
   dropNulls(): Series;
+  /**
```
Comment (universalmind303) [inline polars/series/index.ts:225]: moving this to `shared-trait` will reduce the code duplication between this and expr.

File: __tests__/series.test.ts (hunk 1 above)
Comment (Bidek56) [reply __tests__/series.test.ts:427]: So you want to move 140 tests to `expt.test`?

File: polars/lazy/expr/index.ts (hunk 2 above)
Comment (Bidek56) [reply polars/lazy/expr/index.ts:181]: Should I remove all the mathjax then?

File: polars/series/index.ts (hunk 6 above)
Comment (Bidek56) [reply polars/series/index.ts:225]: I need to think about this, not sure how an interface going to help with 3 different function names.
If you have an example that would be great. Thx

File: __tests__/series.test.ts (hunk 1 above)
Comment (universalmind303) [reply __tests__/series.test.ts:427]: no, just the new ones for ewm

**Review by universalmind303 — APPROVED** [review]

**Pull request merged**